provision.sh

```diff
@@ -41,14 +41,15 @@ echo "-------- Installing NVM & Node --------------"
 echo "---------------------------------------------"
 
 # install nvm
-su - vagrant -c "wget -qO- https://raw.githubusercontent.com/creationix/nvm/v0.31.1/install.sh | bash"
+su - vagrant -c "wget -qO- https://raw.githubusercontent.com/creationix/nvm/v0.31.1/install.sh | bash >/dev/null 2>&1"
 
 # source bash and install node
 su - vagrant <<NVM
 
 source ~/.nvm/nvm.sh
-echo 'sourcing done'
-nvm install 4.4.3
+echo 'Sourcing done'
+nvm install 4.4.3 > /dev/null 2>&1
```

Comment:
No, it pipes the output to `/dev/null`, but any error would come up in output because of `2>&1`..